src/Raven.Client/Documents/Operations/ConnectionStrings/RemoveConnectionStringOperation.cs

```diff
@@ -59,6 +59,9 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
 
     public class RemoveConnectionStringResult
     {
+        [Obsolete("Not used to get operation result. Use RaftCommandIndex instead")]
```

Comment:
`[Obsolete("RemoveConnectionStringResult.ETag is not supported anymore. Will be removed in next major version of the product. Please use RemoveConnectionStringResult.RaftCommandIndex instead.")]`

Reply:
done